src/StoreKeeper/WooCommerce/B2C/PaymentGateway/PaymentGateway.php

```diff
@@ -350,18 +345,15 @@ public static function markRefundAsSynced($orderId, $skRefundId, $refundId): boo
     /**
      * @param $orderId
      * @param $refundId
-     *
-     * @return bool
      */
-    public static function refundExists($orderId, $refundId): ?bool
+    public static function refundExists($orderId, $refundId): ?int
```

Comment:
```suggestion
    public static function refundExists($orderId, $refundId): ?bool
```

Please retest the payments that one could not work correctly